api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,25 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// * HTTP :ref:`router filter <config_http_filters_router>`is
```

Comment:
You can drop this line entirely actually, the entry above is actually file-wide.

Reply:
done